### PR TITLE
update rules

### LINKED ISSUE #190: Enable linter rules that can be automatically fixed
## Overview

The goal of this issue is to run all the rules that can be **automatically** fixed by running `lint --fix`.
The rules should follow https://github.com/telosnetwork/teloscan/issues/127

## Acceptance criteria

- Identify the rules that are automatically fixed from https://github.com/telosnetwork/teloscan/issues/127
- Run the fix
- Open the PR
- Create issues for the ones that can't be automatically fixed from https://github.com/telosnetwork/teloscan/issues/127

## Tasks

- [ ] [Default linting rules](https://github.com/telosnetwork/teloscan/issues/127) are applied
- [ ] #216 is merged
- [ ] All issues have been created

### PR DESCRIPTION
**Pull request by DevArtemNuss**
# Fixes  #190 
https://github.com/telosnetwork/teloscan/issues/127

## Description
Added new rulers for eslint 

   indent: ["error", 4],
    'max-len': ["warn", { "code": 120, "tabWidth": 2 }],
    'no-trailing-spaces' : ["error", { "skipBlankLines": true }],
    eqeqeq: ["error", "always"],   == -> ===
    semi: ["error", "always"],
    'eol-last': ["error", "never"],
    "vue/component-tags-order": ["error", {
      "order": [ [ "script", "template" ], "style" ] 
    }], 

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages

### CODE REVIEW

File: src/pages/accounts/add/SendOtp.vue [hunk 1]
```diff
@@ -7,52 +7,52 @@ import { countriesPhoneCode } from '~/mixins/countries-phone-code';
...
+        ...mapActions('accounts', ['sendOTP']),
+        async onSendOTP() {
+            this.resetValidation(this.form);
+            this.error = null;
+            if (!(await this.validate(this.form))) return;
+            /* const phoneUtil = PhoneNumberUtil.getInstance()
       const number = phoneUtil.parseAndKeepRawInput(`${this.form.countryCode.dialCode}${this.form.smsNumber}`, this.form.countryCode.code)
       this.form.internationalPhone = phoneUtil.format(number, PhoneNumberFormat.INTERNATIONAL)
       this.submitting = true
       const result = await this.sendOTP(this.form) */
```
Comment (donnyquixotic) [inline src/pages/accounts/add/SendOtp.vue:45]: remove this commented code if no longer applicable

File: src/pages/accounts/add/VerifyOtp.vue [hunk 2]
```diff
@@ -5,45 +5,45 @@ import { utils } from '~/mixins/utils';
...
+    methods: {
+        ...mapActions('accounts', ['verifyOTP']),
+        ...mapMutations('general', ['setSuccessMsg']),
+        async onVerifyOTP() {
+            this.resetValidation(this.form);
+            this.error = null;
+            if (!(await this.validate(this.form))) return;
+            this.submitting = true;
+            // const { success, error } = await this.verifyOTP(this.form)
+            // if (success) {
```
Comment (donnyquixotic) [inline src/pages/accounts/add/VerifyOtp.vue:32]: remove commented code

File: src/pages/accounts/add/VerifyOtp.vue [hunk 3]
```diff
@@ -5,45 +5,45 @@ import { utils } from '~/mixins/utils';
...
+            this.$router.push({ path: '/accounts/add/congratulations' });
+            // } else {
+            //  this.error = error
+            // }
```
Comment (donnyquixotic) [inline src/pages/accounts/add/VerifyOtp.vue:36]: remove commented code

File: src/pages/profiles/add/EditImage.vue [hunk 4]
```diff
@@ -97,27 +97,27 @@ export default {
           this.croppa.refresh()
         })
         */
```
Comment (donnyquixotic) [inline src/pages/profiles/add/EditImage.vue:99]: remove commented code

File: src/pages/profiles/list/ContactList.vue [hunk 5]
```diff
@@ -22,60 +22,60 @@ main
...
+            }
+        },
+        async onSearch() {
+            // await this.searchProfiles({ search: this.search, clean: true, lastEvaluatedKey: this.profileList.lastEvaluatedKey })
+            this.clearProfilesList();
+            this.isFirst = true;
+            this.$refs.infiniteScroll.reset();
+            this.$refs.infiniteScroll.resume();
+            this.$refs.infiniteScroll.poll();
+            // v.preventDefault()
```
Comment (donnyquixotic) [inline src/pages/profiles/list/ContactList.vue:76]: remove commented code

File: src/pages/profiles/list/ContactList.vue [hunk 6]
```diff
@@ -22,60 +22,60 @@ main
...
+                    lastEvaluatedKey: this.profileList.lastEvaluatedKey,
+                });
+                this.isLoading = false;
+                done();
+            } else {
+                this.$refs.infiniteScroll.stop();
+            }
+        },
+        async onSearch() {
+            // await this.searchProfiles({ search: this.search, clean: true, lastEvaluatedKey: this.profileList.lastEvaluatedKey })
```
Comment (donnyquixotic) [inline src/pages/profiles/list/ContactList.vue:70]: remove commented code

File: src/pages/profiles/read/ProfileDetail.vue [hunk 7]
```diff
@@ -87,36 +87,36 @@ export default {
       } else return ''
     }
     */
```
Comment (donnyquixotic) [inline src/pages/profiles/read/ProfileDetail.vue:89]: remove commented code

File: src/pages/profiles/read/ProfileDetail.vue [hunk 8]
```diff
@@ -87,36 +87,36 @@ export default {
...
+        if (this.missingProfile) {
+            this.$router.push({ name: 'userRegister' });
+        }
+        this.$store.commit('profiles/setSelectedProfile', []);
 
     /*
     if (!this.$store.getters['profiles/isRegistered']) {
       this.$router.push({ name: 'userRegister' })
     }
     */
```
Comment (donnyquixotic) [inline src/pages/profiles/read/ProfileDetail.vue:108]: remove commented code

File: src/pages/profiles/read/ProfileDetail.vue [hunk 9]
```diff
@@ -87,36 +87,36 @@ export default {
...
-  methods: {
-    ...mapActions('profiles', ['getProfile']),
+    methods: {
+        ...mapActions('profiles', ['getProfile']),
     /*
     goToChat () {
       this.$store.commit('messages/setActiveChat', { activeChat: this.Profile.eosAccount, avatarImage: this.Profile.publicData.avatarImage, s3Identity: this.Profile.publicData.s3Identity })
       this.$router.push({ name: 'chat' })
     }
     */
```
Comment (donnyquixotic) [inline src/pages/profiles/read/ProfileDetail.vue:118]: remove commented code

File: src/pages/trails/ballots/view/BallotView.vue [hunk 10]
```diff
@@ -12,365 +12,365 @@ const FROM_STAKE = 'stake';
...
+            showDetails: false,
+        };
+    },
+    async mounted() {
+        this.getLoggedUserVotes(this.$route.params.id);
+        await this.fetchBallot(this.$route.params.id);
+        window.addEventListener('scroll', this.updateScroll);
+        this.loading = false;
+    },
+    beforeUnmount() {
```
Comment (donnyquixotic) [inline src/pages/trails/ballots/view/BallotView.vue:49]: remove unused lifecycle hook

File: src/pages/works/components/ExpansionProposal.vue [hunk 11]
```diff
@@ -34,102 +34,102 @@ q-expansion-item(
...
-    async onShow() {
-      if (!this.isAuthenticated) {
-        return;
-      }
+    methods: {
+        setYesVotes() {
+            const opts = this.proposal.ballotData.options;
+            const yes = parseFloat(opts[2].value.split(' ')[0]);
+            const no = parseFloat(opts[1].value.split(' ')[0]);
+            // const total = parseFloat(this.proposal.ballotData.total_raw_weight.split(' ')[1])
```
Comment (donnyquixotic) [inline src/pages/works/components/ExpansionProposal.vue:75]: remove unused var

File: src/router/routes.js [hunk 12]
```diff
@@ -1,149 +1,149 @@
...
-    meta: { guest: false },
-  },
+    {
+        path: '/login',
+        component: () => import('pages/login/LoginPage.vue'),
+        meta: { layout: 'guest', title: 'pages.login.title', guest: true },
+    },
+    // { path: '/accounts/add', component: () =>
+    // import('pages/accounts/add/SendOtp.vue'),
+    // meta: { layout: 'guest', title: 'pages.accounts.add.title', guest: true } },
```
Comment (donnyquixotic) [inline src/router/routes.js:9]: remove commented code

File: src/router/routes.js [hunk 13]
```diff
@@ -1,149 +1,149 @@
...
+    {
+        path: '/accounts/add',
+        component: () => import('pages/accounts/add/CreateAccount.vue'),
+        meta: { layout: 'guest', title: 'pages.accounts.add.title', guest: true },
+    },
+    // { path: '/accounts/add/verifyOTP', component:
+    // () => import('pages/accounts/add/VerifyOtp.vue'),
+    // meta: { layout: 'guest', title: 'pages.accounts.add.title', guest: true } },
```
Comment (donnyquixotic) [inline src/router/routes.js:17]: remove commented code

File: src/store/module-example/actions.js [hunk 14]
```diff
@@ -1 +1 @@
-export function someAction(/* context */) {}
+export function someAction(/* context */) {}
```
Comment (donnyquixotic) [inline src/store/module-example/actions.js:1]: remove `module-example` directory

File: src/store/module-example/getters.js [hunk 15]
```diff
@@ -1 +1 @@
-export function someGetter(/* state */) {}
+export function someGetter(/* state */) {}
```
Comment (donnyquixotic) [inline src/store/module-example/getters.js:1]: remove `module-example` directory

File: src/store/module-example/index.js [hunk 16]
```diff
@@ -4,9 +4,9 @@ import * as mutations from './mutations';
...
-  mutations,
-  actions,
-  state,
-};
+    namespaced: true,
+    getters,
+    mutations,
+    actions,
+    state,
+};
```
Comment (donnyquixotic) [inline src/store/module-example/index.js:12]: remove `module-example` directory

File: src/store/module-example/mutations.js [hunk 17]
```diff
@@ -1 +1 @@
-export function someMutation(/* state */) {}
+export function someMutation(/* state */) {}
```
Comment (donnyquixotic) [inline src/store/module-example/mutations.js:1]: remove `module-example` directory

File: src/store/module-example/state.js [hunk 18]
```diff
@@ -1,5 +1,5 @@
 export default function () {
-  return {
+    return {
     //
-  };
-}
+    };
+}
```
Comment (donnyquixotic) [inline src/store/module-example/state.js:5]: remove `module-example` directory

File: src/store/notifications/actions.js [hunk 19]
```diff
@@ -1,4 +1,4 @@
 /*
 export function someAction (context) {
 }
-*/
+*/
```
Comment (donnyquixotic) [inline src/store/notifications/actions.js:4]: remove unused file

File: src/store/profiles/actions.js [hunk 20]
```diff
@@ -1,139 +1,139 @@
 // import PPP from '@smontero/ppp-client-api'
```
Comment (donnyquixotic) [inline src/store/profiles/actions.js:1]: remove commented import

File: src/store/profiles/actions.js [hunk 21]
```diff
@@ -1,139 +1,139 @@
...
-    commit('setProfiles', { lastEvaluatedKey: lastKey, items: filteredProfiles })
-  /*
+        const lastKey = profileResult.rows ? profileResult.rows[profileResult.rows.length - 1] : undefined;
+        commit('setProfiles', { lastEvaluatedKey: lastKey, items: filteredProfiles });
+        /*
   await profileApi.searchProfiles(search, limit, lastEvaluatedKey).then(response => {
     if (clean === true) commit('clearProfilesList')
     commit('setProfiles', response)
   })
   */
```
Comment (donnyquixotic) [inline src/store/profiles/actions.js:102]: remove commented code

File: src/store/profiles/actions.js [hunk 22]
```diff
@@ -147,4 +147,4 @@ export const verifyEmail = async function ({ commit }, code) {
 export const setPPPLoading = async function ({ commit }, PPPLoading) {
   commit('setPPPLoading', PPPLoading)
 }
```
Comment (donnyquixotic) [inline src/store/profiles/actions.js:149]: remove commented code

File: src/store/profiles/getters.js [hunk 23]
```diff
@@ -1,30 +1,30 @@
...
-  // if (profile.commPref === 'EMAIL') {
-  //   return !!profile.emailInfo.needsToVerify
-  // } else if (profile.commPref === 'SMS') {
-  //   return !!profile.smsInfo.needsToVerify
-  // }
+    // if (profile.commPref === 'EMAIL') {
+    //   return !!profile.emailInfo.needsToVerify
+    // } else if (profile.commPref === 'SMS') {
+    //   return !!profile.smsInfo.needsToVerify
+    // }
```
Comment (donnyquixotic) [inline src/store/profiles/getters.js:14]: remove commented code

File: src/store/testnet/getters.js [hunk 24]
```diff
@@ -1,4 +1,4 @@
 /*
 export function someGetter (state) {
 }
-*/
+*/
```
Comment (donnyquixotic) [inline src/store/testnet/getters.js:4]: remove unused file

File: src/store/testnet/mutations.js [hunk 25]
```diff
@@ -1,4 +1,4 @@
 /*
 export function someMutation (state) {
 }
-*/
+*/
```
Comment (donnyquixotic) [inline src/store/testnet/mutations.js:4]: remove unused file

File: src/store/testnet/state.js [hunk 26]
```diff
@@ -1,3 +1,3 @@
 export default {
-  //
-};
+    //
+};
```
Comment (donnyquixotic) [inline src/store/testnet/state.js:3]: remove unused export/file

File: src/store/transfers/getters.js [hunk 27]
```diff
@@ -1,4 +1,4 @@
 /*
 export function someGetter (state) {
 }
-*/
+*/
```
Comment (donnyquixotic) [inline src/store/transfers/getters.js:4]: remove unused file

File: src/store/transfers/mutations.js [hunk 28]
```diff
@@ -1,4 +1,4 @@
 /*
 export function someMutation (state) {
 }
-*/
+*/
```
Comment (donnyquixotic) [inline src/store/transfers/mutations.js:4]: remove unused file

File: src/store/transfers/state.js [hunk 29]
```diff
@@ -1,3 +1,3 @@
 export default {
-  //
-};
+    //
+};
```
Comment (donnyquixotic) [inline src/store/transfers/state.js:3]: remove unused file

File: src/store/validators/actions.js [hunk 30]
```diff
@@ -5,69 +5,69 @@ export function someAction (context) {
...
+        after: moment
+            .utc()
+            .subtract(days, 'days')
+            .format('YYYY-MM-DDTHH:mm:ss.SSS[Z]'),
+        limit: 1000,
+        sort: 'asc',
+    };
+    let bpMap = {};
+    while (haveMore) {
     // params.before = moment.utc().format('YYYY-MM-DDTHH:mm:ss.SSS[Z]')
```
Comment (donnyquixotic) [inline src/store/validators/actions.js:25]: remove commented code

File: src/store/validators/actions.js [hunk 31]
```diff
@@ -5,69 +5,69 @@ export function someAction (context) {
...
+        for (let i = 0; i < acts.length; i++) {
+            if (acts[i].global_sequence > biggest) {
+                biggest = acts[i].global_sequence;
+            }
+            const action = acts[i];
+            const producer = action.producer;
+            const cpuUs = action.cpu_usage_us;
+            const timestamp = action.timestamp;
+            // const block = action.block_num
+            // const trxId = action.trx_id
```
Comment (donnyquixotic) [inline src/store/validators/actions.js:48]: remove unused vars

File: src/store/validators/getters.js [hunk 32]
```diff
@@ -1,4 +1,4 @@
 /*
 export function someGetter (state) {
 }
-*/
+*/
```
Comment (donnyquixotic) [inline src/store/validators/getters.js:4]: remove unused file

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

**Review by donnyquixotic — APPROVED** [review]

**Review by donnyquixotic — CHANGES_REQUESTED** [review]
still pending changes requested

**Review by donnyquixotic — APPROVED** [review]

File: .eslintrc.js [hunk 33]
```diff
@@ -73,8 +73,16 @@ module.exports = {
...
-    // allow debugger during development only
+    indent: ["error", 4],
+    'max-len': ["warn", { "code": 120, "tabWidth": 2 }],
+    'no-trailing-spaces' : ["error", { "skipBlankLines": true }],
+    eqeqeq: ["error", "always"],
+    semi: ["error", "always"],
+    'eol-last': ["error", "never"],
+    "vue/component-tags-order": ["error", {
+      "order": [ [ "script", "template" ], "style" ]
+    }],
```
Comment (EJSG) [inline .eslintrc.js:84]: single/double should be consistent for the rules, eg. `'eol-last'` vs `"vue/component-tags-order"` (which I guess will take care of itself when a quotes rules is added haha)

**Review by EJSG — APPROVED** [review]
approved, looks good. optionally you can address my one comment about quote style consistency but that doesnt need to block merging

File: .eslintrc.js (hunk 33 above)
Comment (Viterbo) [reply .eslintrc.js:84]: Somehow it didn't !!
So I fixed it personally for you (5b303e0c8e18f08103083a1578d4b9768750504e)

**Review by Viterbo — APPROVED** [review]

**Comment by Kersten-TCD** [comment]
@Viterbo I found an issue when clicking on Elections in top menu, this will not refresh the page and proposals are still shown. Same happens when selecting Proposals after refresh on elections page. I need to refresh page to see proposals again.

Will check if this can be reproduced on PR #227 once this PR is merged.

**Pull request merged**